StRoot/StDbUtilities/StMagUtilities.cxx

```diff
@@ -2906,6 +2924,117 @@ void StMagUtilities::UndoSpaceChargeR2Distortion( const Float_t x[], Float_t Xpr
 
   
 //________________________________________
+
+
+/// FixedTarget 1/r SpaceCharge Distortion
+/*!
+  Similar to R2 distortion correction for SpaceCharge, but using distinct radiual distributions on
+  the east and west sides as determined by studies of URQMD simulations of fixed target
+  collisions conducted by Yue-Hang Leung in June 2022.
+*/
+void StMagUtilities::UndoSpaceChargeFXTDistortion( const Float_t x[], Float_t Xprime[] , Int_t Sector )
+{
+  
+  const Int_t     ORDER       =    1 ;  // Linear interpolation = 1, Quadratic = 2
```

Comment:
per STAR naming conventions, prefer `int` to `Int_t`, `double` to `Double_t`, etc...